app/models/api/v3/context.rb

```diff
@@ -19,6 +19,25 @@ class Context < BlueTable
       validates :country, presence: true
       validates :commodity, presence: true, uniqueness: {scope: :country}
 
+      def is_visible?
```

Comment:
this seems to be slightly out of scope. could you explain how it works, please?

Reply:
Sure, so we mentioned that each context should have an `EXPORTER` and `COUNTRY` node in order to be able to render the world map properly. We'll add checks in the validation service to detect this, but it will still be possible to have such contexts in the database. If it so happens, they should be suppressed from the `/contexts` response, and this method does a rough check for whether or not it's good to show a particular context. I guess when the validation service is merged it will be possible to replace it with a more succinct check e.g. `all_checks_passing` which checks for even more things, but in the meantime I only included this minimum to satisfy the requirements of the world map.

Reply:
ok, so this would be a system-wide setting, and not something limited to the world map endpoint, right? My concern here is that this might be perceived as a fully functional "hide this context in the whole site" flag, which I'm not sure we want to have/support.

Reply:
Maybe this is not the most clever way, but I definitely think we need a way to disable a context from being active system-wide if we have reason to expect it will break things in any of the parts of the site. For example, if it doesn't have the EXPORTER or COUNTRY node the sankey will likely break as well as the world map. I could remove this check from this PR and add a more robust version in the validation PR.